src/state/_common/selectors.js

```diff
@@ -740,33 +735,42 @@ const getUsesWithActiveDependency = (getSubstate) => {
 
 const getStateToSave = (getSubstate) => {
 	return (state) => {
-		let selectedState = {};
-		if (getSubstate(state).activeKey) {
-			selectedState.activeKey = getSubstate(state).activeKey;
-		} else if (getSubstate(state).activeKeys) {
-			selectedState.activeKeys = getSubstate(state).activeKeys;
+		const activeKey = getSubstate(state).activeKey;
```

Comment:
Is there any advantage of this notation?

Reply:
I can revert if you want.

The difference is that old version mutates `selectedState` while newer version does not. I usually try to avoid mutations (doesn't really matter much if they're encapsulated inside a functions and doesn't affect world around).

Also older version calls `getSubState` one less time - should really call it once, but I guess I wasn't thinking much when modifying it (but since I suppose that inside is just property access and not some heavy computation it doesn't really matter). Which is another thing I usually do - avoid calling functions/methods more than necessary without having to think if it makes a difference in particular case.

So in this particular case there doesn't seems to be any advantage. It's just some rules I usually follow so that I don't have to think much about particular cases.